src/app/modules/dashboard/pages/dashboard/dashboard.component.spec.ts

```diff
@@ -0,0 +1,36 @@
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+import {NoopAnimationsModule} from '@angular/platform-browser/animations';
+import {RouterTestingModule} from '@angular/router/testing';
+import {HttpClientTestingModule} from '@angular/common/http/testing';
+
+import {DashboardComponent} from './dashboard.component';
+import {MaterialModule} from '../../../../shared/material.module';
+import {LayoutModule} from '../../../../shared/layout/layout.module';
+
+describe('DashboardComponent', () => {
+    let component: DashboardComponent;
+    let fixture: ComponentFixture<DashboardComponent>;
+
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
+            declarations: [DashboardComponent],
```

Comment:
You need to declare the children components being used in the `DashboardComponent` for this test to work.
```suggestion
            declarations: [DashboardComponent, CpuStatsComponent, ...],
```